packages/env/lib/commands/start.js

```diff
@@ -26,6 +26,8 @@ const {
 	configureWordPress,
 	setupWordPressDirectories,
 } = require( '../wordpress' );
+const { didMetaChange, setMeta } = require( '../meta' );
```

Comment:
This naming kind of obscures what this is. It's not a metafile; it's more like a cache file or a hash list.

Reply:
I tried to deliberately generalize it into a metafile -- we can `setMeta` to whatever value you want and then retrieve that same value later. In the case of `start` that value happens to be a hash of the config file

But I'm happy to change the name to something more like "cache". That could work in both ways, and also drives home the point that it's "temporary" (i.e. everything should still work if the cache is not set)